src/views/forms/UnstakeForm/index.tsx

```diff
@@ -133,10 +143,14 @@ export const UnstakeForm = ({ onDone, className }: UnstakeFormProps) => {
     } finally {
       setIsLoading(false);
     }
-  }, [isAmountValid, amounts, undelegate, onDone]);
+  }, [isAmountValid, amounts, undelegate, onDone, totalAmount]);
 
   const onChangeAmount = useCallback((validator: string, value: number | undefined) => {
     setAmounts((a) => ({ ...a, [validator]: value }));
+    track(AnalyticsEvent.UnstakeInput, {
```

Comment:
hmm I wonder if we should debounce the change amount if we're logging here

Reply:
I spoke to Yang about it when I added this, and he was ok with tracking it like this. But I'll look into debouncing the tracking specifically